### DISCUSSION TITLE
No sound on *.avi file

### DISCUSSION THREAD

**Discussion by sobuj53** [spencerwooo/onedrive-vercel-index#438] [Q&A]
Hello I'm trying to play an *.avi file from my deployed index but there is no sound even though the video plays just fine with the DPlayer. When I play the file locally the everything works just fine. Can you please take a look?
Example file link, [Minamata.2021](https://www.sobuj53.cf/Minamata.2021.HDRip.XviD.AC3-EVO%5BTGx%5D/Minamata.2021.HDRip.XviD.AC3-EVO.avi)

**Comment by spencerwooo** [comment]
Not all video format is supported by DPlayer. :(

**Comment by sobuj53** [comment]
Some other video formats is also not playing for example *.mkv [Wrath_of_Man](https://www.sobuj53.tk/Wrath%20of%20Man%20(2021)%20%2B%20Extras%20(1080p%20WEB-DL%20x265%20HEVC%2010bit%20EAC3%205.1%20SAMPA)/Wrath%20of%20Man%20(2021)%20(1080p%20WEB-DL%20x265%20SAMPA).mkv). However the other gdrive index of mine by [ParveenBhadoo](https://gitlab.com/ParveenBhadooOfficial/Google-Drive-Index) which uses [Plyr](https://github.com/sampotts/plyr) works just fine. Would you consider checking it out?

**Reply by spencerwooo** [reply]
Oh, thanks for the `Plyr` library suggestion, I didn't know about this before and thought DPlayer was the best-in-slot one as for video players. I'll definitely check this one out.

**Comment by zkeq** [comment]
You provide the address work on the Edge of the browser..

https://user-images.githubusercontent.com/62864752/154038768-a72e32bc-6e3a-4623-be38-e196e5d5320d.mp4

**Reply by sobuj53** [reply]
Thank you for checking out. But it didn't work on my Chrome Version 98.0.4758.102 (Official Build) (64-bit). The video played just fine but there was no audio. But for the later file, both audio and video have failed to load.
